instagram/src/App.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import moment from "moment";
+import dummyData from "./dummy-data";
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from "./components/PostContainer/PostContainer";
+import "./css/main.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      newCommentText: "",
+      searchText: "",
+      liked: {}
+    };
+  }
+
+  componentDidMount() {
+    // There might be a more succinct way to do this, but the following turns the timestamp string from dummyData into Unix Epoch Time (while leaving the other data alone). Earlier I had just edited dummy-data.js, so this is more elegant.
+    const data = dummyData.map(post => {
+      return (post = {
+        username: post.username,
+        visible: true,
+        thumbnailUrl: post.thumbnailUrl,
+        imageUrl: post.imageUrl,
+        likes: post.likes,
+        timestamp: moment(post.timestamp, "MMMM Do YYYY, h:mm:ss a").valueOf(),
```

Comment:
cleaner way that utilizes moment's syntax instead of editing the source data, agreed